src/app.js

```diff
@@ -1 +1,238 @@
-console.log('Hallo daar!');
\ No newline at end of file
+console.log('Hallo daar!');
+import axios from "axios";
+
+
+
+
+//------------------------------------------------------------------------OPDRACHT 1--------------------------------//
+// koppeling aan index
+const landName = document.getElementById("list-Landen")
+
+
+async function banaan( ) {
+
+    try {
+        const response = await axios.get( 'https://restcountries.com/v2/all');
+        // console.log(response);
+
+        landName.replaceChildren();
+
+        response.data.sort ((a,b) => (a.population - b.population));
+
+        //door de data heen mappen
+        response.data.map((country) => {
```

Comment:
Je zou hier nog gebruik kunnen maken van spreading, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
        response.data.map(({ name, population, region, flags }) => {
```